modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in all namespaces. After the objects are removed, you can then remove the Operator and its namespace.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The File Integrity Operator is installed.
+
+.Procedure
+
+. Navigate to the *Operators* -> *Installed Operators* -> File Integrity Operator page.
+
+. Use the default option *Show operands in: All namespaces* to list all `FileIntegrity` objects in all namespaces.
+
+. Click the Options menu {kebab} and use it to delete all `FileIntegrity` objects.
+
+. Switch to the *Administration* -> *Operators* -> *Installed Operators* page.
```

Comment:
```suggestion
. Go to the *Administration* -> *Operators* -> *Installed Operators* page.
```

![Screenshot From 2025-04-09 11-12-40](https://github.com/user-attachments/assets/b4fa4aa9-8179-41b9-8318-41fac715ccf4)

Reply:
Maybe it's something on my end but I do not see such an option: 

![Screenshot From 2025-04-09 11-14-55](https://github.com/user-attachments/assets/a0597edd-f093-46fc-93b7-aeeeef00a750)


Maybe a tip or note is needed to explain why the option could be missing?

Reply:
Good catch. 
Perhaps you meant the Administrator perspective?